scripts/install

```diff
@@ -190,7 +72,8 @@ ynh_systemd_action --service_name=$app --action=start --log_path=systemd --line_
 ynh_script_progression --message="Create the first administrator and team..." --weight=1
 
 team_name=$(echo "$team_display_name" | iconv -f utf8 -t ascii//TRANSLIT//IGNORE | sed -e 's/[^[:alnum:]]/-/g' | tr -s '-' | tr A-Z a-z)
-bin_mmctl="$final_path/bin/mmctl"
+bin_mmctl="$install_dir/bin/mmctl"
+email=$(ynh_user_get_info --username=$admin --key=mail)
 
 # mmctl is not packaged with ARM versions yet
 if [[ -f "$bin_mmctl" ]]; then
```

Comment:
`mmctl` is now packaged with all arm releases (first-party or third-party): this sanity check can be removed.